src/main/java/stirling/software/SPDF/controller/api/SplitPdfBySectionsController.java

```diff
@@ -145,4 +156,23 @@ public List<PDDocument> splitPdfPages(
 
         return splitDocuments;
     }
+
+    public ByteArrayOutputStream mergeSplitPdfs(List<PDDocument> splitDocuments)
+            throws IOException {
+        PDFMergerUtility pdfMerger = new PDFMergerUtility();
+        ByteArrayOutputStream mergedPdfOutputStream = new ByteArrayOutputStream();
+
+        for (PDDocument doc : splitDocuments) {
+            String tempFileName = UUID.randomUUID().toString();
+            File tempFile = File.createTempFile(tempFileName, ".pdf");
+            doc.save(tempFile);
+            pdfMerger.addSource(tempFile);
+            tempFile.deleteOnExit();
+        }
+
+        pdfMerger.setDestinationStream(mergedPdfOutputStream);
+        pdfMerger.mergeDocuments(null);
+
+        return mergedPdfOutputStream;
+    }
```

Comment:
Can we reuse the implementation in `MergeController`?

Reply:
Okay, I will revise it again